src/Form/__tests__/FieldSelect-test.js

```diff
@@ -0,0 +1,129 @@
+jest.dontMock('../FieldSelect');
+jest.dontMock('../icons/IconEdit');
+jest.dontMock('../../Util/KeyboardUtil');
+
+/* eslint-disable no-unused-vars */
+var React = require('react');
+/* eslint-enable no-unused-vars */
+var TestUtils = require('react-addons-test-utils');
+
+var FieldSelect = require('../FieldSelect');
+
+describe('FieldSelect', function () {
```

Comment:
Let's add a test for this: https://github.com/mesosphere/reactjs-components/pull/260#issuecomment-221135945